parachains/runtimes/collectives/collectives-polkadot/src/ambassador/tracks.rs

```diff
@@ -43,17 +48,17 @@ impl pallet_referenda::TracksInfo<Balance, BlockNumber> for TracksInfo {
 
 	/// Return the array of available tracks and their information.
 	fn tracks() -> &'static [(Self::Id, pallet_referenda::TrackInfo<Balance, BlockNumber>)] {
-		static DATA: [(TrackId, pallet_referenda::TrackInfo<Balance, BlockNumber>); 4] = [
+		static DATA: [(TrackId, pallet_referenda::TrackInfo<Balance, BlockNumber>); 9] = [
 			(
-				constants::CANDIDATE,
+				constants::AMBASSADOR_TIER_1,
 				pallet_referenda::TrackInfo {
-					name: "candidate",
-					max_deciding: 50,
-					decision_deposit: 10 * UNITS,
+					name: "ambassador tier 1",
+					max_deciding: 10,
+					decision_deposit: 5 * DOLLARS,
 					prepare_period: 30 * MINUTES,
 					decision_period: 7 * DAYS,
 					confirm_period: 30 * MINUTES,
```

Comment:
Is there need for a short confirmation period? Would 24h not be fine? 30 mins is nothing really.